frontend/public/style/_overrides.scss

```diff
@@ -223,138 +217,24 @@ h6 {
-    --pf-c-nav__simple-list-link--active--Color: var(--pf-c-nav__list-link--hover--Color);
-    --pf-c-nav__simple-list-link--Color: var(--pf-c-nav__list-link--Color);
-    --pf-c-nav__simple-list-link--focus--BackgroundColor: var(--pf-c-nav__list-link--hover--BackgroundColor);
-    --pf-c-nav__simple-list-link--focus--Color: var(--pf-c-nav__list-link--hover--Color);
-    --pf-c-nav__simple-list-link--focus--FontWeight: var(--pf-global--FontWeight--normal);
-    --pf-c-nav__simple-list-link--hover--BackgroundColor: var(--pf-c-nav__list-link--hover--BackgroundColor);
-    --pf-c-nav__simple-list-link--hover--Color: var(--pf-c-nav__list-link--hover--Color);
-    --pf-c-nav__simple-list-link--m-current--BackgroundColor: var(--pf-c-nav__list-link--hover--BackgroundColor);
-    --pf-c-nav__simple-list-link--m-current--Color: var(--pf-c-nav__simple-list-link--active--Color);
-    --pf-c-nav__simple-list-link--PaddingBottom: 2px;
-    --pf-c-nav__simple-list-link--PaddingRight: 5px;
-    --pf-c-nav__simple-list-link--PaddingTop: 2px;
-    // Subnav
-    --pf-c-nav__subnav--MarginTop: 0;
-    --pf-c-nav__subnav--MaxHeight: 100%;
-    // Toggle
-    --pf-c-nav__list-toggle--PaddingRight: 0;
     // Nav Section
     font-size: $co-side-nav-font-size;
-  }
 
-  .pf-c-nav__item .pf-c-nav__simple-list .pf-c-nav__separator {
-    --pf-c-nav__separator--MarginLeft: var(--pf-c-nav__list-link--PaddingLeft);
-    margin-right: var(--pf-c-nav__list-link--PaddingRight);
-
-    @media screen and (min-width: $grid-float-breakpoint) {
-      --pf-c-nav__separator--MarginLeft: var(--pf-c-nav__list-link--md--PaddingLeft);
-      margin-right: var(--pf-c-nav__list-link--md--PaddingRight);
-    }
-  }
-
-  .pf-c-nav__list {
-    list-style: none; // turn off list-styles to fix bug in Edge
-    > .pf-c-nav__item {
-      border-bottom: 1px solid #{$color-pf-black};
-      margin-top: 0;
-
-      &:not(.pf-m-current) > .pf-c-nav__link {
-        &:active,
-        &:focus,
-        &:hover {
-          background-color: var(--pf-c-nav__list-link--hover--BackgroundColor);
-        }
-      }
-
-      &.pf-m-current {
-        background-color: #{$color-pf-black-800};
-
-        &.pf-m-expanded > .pf-c-nav__link::after {
-          display: none;
-        }
-      }
-
-      &.pf-m-expanded,
-      .pf-c-nav__link:hover {
-        .pf-c-nav__toggle {
-          display: block;
-        }
-      }
+    .pf-c-nav__simple-list .pf-c-nav__link {
+      font-size: $font-size-base;
     }
 
-    .pf-c-nav__link {
-      align-items: center;
-      padding-left: var(--pf-c-nav__list-link--PaddingLeft);
-
-      @media screen and (min-width: $grid-float-breakpoint) {
-        padding-left: var(--pf-c-nav__list-link--md--PaddingLeft);
-      }
-
-      &.pf-m-current {
-        background-color: var(--pf-c-nav__list-link--hover--BackgroundColor);
-      }
+    // override list styles, necessary due to setting $pf-global--enable-reset to false
```

Comment:
Can this not be set on the list itself rather than the individual bullets?

Reply:
Changed to be on the list, thanks.

Reply:
Any reason to not use `.pf-c-nav__simple-list` as the selector instead of the generic element?

Reply:
I would have to set it on both `.pf-c-nav__simple-list` and `.pf-c-nav__list` but I could do that. You can't see them due to margins (except on the separator, but they are there.